### DISCUSSION TITLE
EuroPC 1.44 Floppy

### DISCUSSION THREAD

**Discussion by solemgar** [86Box/86Box#3422] [Q&A] [answered]
This is probably me failing to remember basic DOS stuff. I am using the europc rom for nostalgia purposes. So far everything works fine, hdd, video etc, but I can only read 720k imgs, none of the 1.44 ones are readable (general error) no matter how many combinations of 1.44 floppy and controllers I try. I am using the original dos 3.30 boot drive.

**Comment by waltje** [comment, marked as answer]
The EuroPC had no BIOS support for 1.44, it's an XT, not an AT+.
So, it will only work if you use a floppy controller that has its own BIOS.  

That said, such controllers did not work in the machine as it did not havea bus, so there is a good chance its BIOS will not scan the bus for anyBIOS extensions..
Fred

    On Friday, June 23, 2023 at 02:23:59 PM EDT, solemgar ***@***.***> wrote:  
 
 


This is probably me failing to remember basic DOS stuff. I am using the europc rom for nostalgia purposes. So far everything works fine, hdd, video etc, but I can only read 720k imgs, none of the 1.44 ones are readable (general error) no matter how many combinations of 1.44 floppy and controllers I try. I am using the original dos 3.30 boot drive.

**Reply by solemgar** [reply]
Oh that's true! I knew it had to be something architectural. I feel a bit ashamed now for not remembering :P It has been so long since I had the europc I completely forgot. 86box it has been quite the discovery, really having a blast!

**Comment by solemgar** [comment]
As a follow up, the main issue I am having is that I can't find versions of some programs/games on 720k floppy, so I can't put them in the emulated box. Is there any way to maybe create a compatible hdd image with the data already in it?

**Comment by waltje** [comment]
Yes, if you are on Windows, use something like 'winimage' to create a 720K image,then open a 1.44M with those files, and start copying between the two.
I am sure there is a way for Linux/macOS too.
Also, you can set up a DOS-based AT machine in 86Box to do this.. give it one360K/1.2M floppy, and one 720/1.44M floppy, and voila.


    On Friday, June 23, 2023 at 03:10:11 PM EDT, solemgar ***@***.***> wrote:  
 
 


As a follow up, the main issue I am having is that I can't find versions of some programs/games on 720k floppy, so I can't put them in the emulated box. Is there any way to maybe create a compatible hdd image with the data already in it?

**Comment by solemgar** [comment]
Thanks! That's the approach I was using but sadly some of the files are +1MB so I can't drop them between images. I'll have to resort to good old arj it seems to split it in several parts :)

**Comment by waltje** [comment]
That, or use a hard disk in that AT machine. WinImage can open those, too.

    On Friday, June 23, 2023 at 03:17:09 PM EDT, solemgar ***@***.***> wrote:  
 
 


Thanks! That's the approach I was using but sadly some of the files are +1MB so I can't drop them between images. I'll have to resort to good old arj it seems to split it in several parts :)

**Comment by solemgar** [comment]
Ah! That's a beautiful idea ! I will try to use a hard drive on a AT machine and do the switcheroo, thanks! Will come back in a bit with the results

**Comment by solemgar** [comment]
@waltje That did the trick. Really appreciate the help! I didn't even had to change anything, just set it as AT and it booted from the same hard drive. I can copy the files now happily . Thank you so much!!!!
